hopsworks-common/src/main/java/io/hops/hopsworks/common/agent/AgentController.java

```diff
@@ -367,26 +363,6 @@ public String listCondaEnvironment(String project) {
         continue;
       }
 
-      //Special case for tensorflow
-      if (libraryName.equals("tensorflow") || libraryName.equals("tensorflow-gpu") || libraryName.equals("tensorflow" +
```

Comment:
@robzor92 I'm just curious, how does TF 1.15 motivate this change?

Reply:
We are not going to install "tensorflow-gpu" as the "tensorflow" package now contains both GPU and CPU support in 1.15.0. Similarily for pytorch we don't install "pytorch-cpu" and "pytorch" anymore, only "pytorch". This means we can get rid of the possibility to install libraries on different machine types (GPU, CPU, ALL) as it is not needed anymore, that will be removed as part of the dockerization effort. For ROCm we are temporarily dropping support for it until we figure out a way to get it in hadoop 3.2 which is why it is also removed here.